README.md

```diff
@@ -24,42 +24,72 @@ Or to install from the latest commit in our repository, run
 cargo install --git https://github.com/EFForg/apkeep.git
 ```
 
+Docker images are also available through the GitHub Container Registry. Aside from using a
+specific release version, the following floating tags are available:
+
+- stable: tracks the latest stable release (recommended)
+- latest: tracks the latest release, including pre-releases
+- edge: tracks the latest commit
+
 ## Usage
 
 See [`USAGE`](https://github.com/EFForg/apkeep/blob/master/USAGE).
 
+For the Docker image the same flags apply, except that the output path is set to /output. So
+make sure to have a volume mounted to /output.
+
 ## Examples
 
 The simplest example is to download a single APK to the current directory:
 
 ```shell
+# Local
 apkeep -a com.instagram.android .
+
+# Docker
+docker run --rm -v output_path:/output ghcr.io/efforg/apkeep:stable -a com.instagram.android
 ```
 
 This downloads from the default source, APKPure, which does not require credentials.  To
 download directly from the google play store:
 
 ```shell
+# Local
 apkeep -a com.instagram.android -d google-play -u 'someone@gmail.com' -p somepass .
+
+# Docker
+docker run --rm -v output_path:/output ghcr.io/efforg/apkeep:stable -a com.instagram.android -d google-play -u 'someone@gmail.com' -p somepass
```

Comment:
I'd like to keep the examples simple.  I would be fine with adding a single line at the bottom or top of the usage segment to just say something like

"To run with `docker`, just replace the `apkeep` at the beginning of these commands with `docker run --rm -v output_path:/output ghcr.io/efforg/apkeep:stable`

Reply:
Also, you'll want to add these lines to the `src/main.rs` header documentation, since `README.md` is auto-generated from that.